backend/editor/entries.py

```diff
@@ -177,28 +177,30 @@ def update_nodes(label, entry, new_node_keys):
             continue
         query.append(f"""\nREMOVE n.{key}\n""")
 
-    #Normalising Node to be Updated
-    normalised_new_node_keys = {}
+    # Adding normalized tags ids corresponding to entry tags
+    normalised_new_node_key = {}
     for keys in new_node_keys.keys():
-        if keys.startswith("tags") and not keys.endswith("str"):
-            if "ids" not in keys:
+        if keys.startswith("tags_") and not keys.endswith("_str"):
+            if "_ids_" not in keys:
                 keys_language_code = keys.split('_', 1)[1]
                 normalised_value = []
                 for values in new_node_keys[keys]:
                     normalised_value.append(normalizing(values, keys_language_code))
-                normalised_new_node_keys[keys] = new_node_keys[keys]
-
-                normalised_new_node_keys["tags_ids_"+keys_language_code] = normalised_value
+                normalised_new_node_key[keys] = normalised_value
+                normalised_new_node_key["tags_ids_"+keys_language_code] = normalised_value
+            else:
+                pass

```

Comment:
The comment was important in my proposal :-) because it explains the "why" (the code tell the how, the comments tells the why, or summarize the what)

```suggestion
                pass   # we generate tags_ids, and ignore the one sent
```